tests/sentry/tasks/integrations/github/test_pr_comment.py

```diff
@@ -348,6 +348,8 @@ def setUp(self):
     @responses.activate
     def test_comment_workflow(self, mock_metrics, mock_issues):
         groups = [g.id for g in Group.objects.all()]
+        titles = [g.title for g in Group.objects.all()]
+        culprits = [g.culprit for g in Group.objects.all()]
```

Comment:
nit: I think you can do this
```suggestion
        groups, titles, culprits = [(g.id, g.titles, g.culprits) for g in Group.objects.all()]
```

Reply:
hmm this doesn't work, but i appreciate the effort c:

Reply:
hmm strange, worked in the online compiler I tried

Reply:
i get `ValueError: not enough values to unpack (expected 3, got 2)`

Reply:
I think I misspelled it. What about
```
groups, titles, culprits = [(g.id, g.title, g.culprit) for g in Group.objects.all()]
```

Reply:
lol i fixed it to that, but i get that error

Reply:
hmm this is pretty interesting, I don't know how the tuple could only contain 2 things when there's clearly 3

Reply:
I think it's prob the title field because it's the only property that's not an actual model field

Reply:
hm yeah it's a property